java/src/jmri/jmrix/openlcb/Bundle.properties

```diff
@@ -37,7 +37,7 @@ OlcbClockAlt2 = Alternate Clock 2
 # {0} will be a node ID like 05:02:01:00:00:F0
 OlcbClockCustom = Custom Clock {0}
 
-AddTurnoutEntryToolTip = <html>Pair of 8 byte events, Thrown;Closed<br>02.01.57.00.00.54.00.72;02.01.57.00.00.54.00.73</html>
+AddTurnoutEntryToolTip = <html>Pair of 8 byte events, Thrown;Closed<br>02.01.57.00.00.54.00.72;02.01.57.00.00.54.00.73<br>Bridged DCC Turnouts: T123</html>
```

Comment:
I would suggest that you add something like "Or ..."

`<br>Or you can use bridged DCC Turnouts: T123</html>`